core/src/main/scala/cats/mtl/ApplicativeAsk.scala

```diff
@@ -1,5 +1,8 @@
-package cats
-package mtl
+package cats.mtl
+
+import cats.Applicative
+
+import acyclic.skipped
```

Comment:
What does this do? Do we need it? :)

Reply:
Result of `scalacOptions += "-P:acyclic:force"`